src/scripts/origins.js

```diff
@@ -22,11 +22,11 @@ export default {
   },
   'atlassian.com': {
     url: '*://*.atlassian.com/*',
-    name: 'Atlassian'
+    name: 'Atlassian / Jira'
```

Comment:
This broke `TogglButton.extractDomain` for me it wanted to resolve the script name as

![image](https://user-images.githubusercontent.com/1716853/57826446-f935f580-77c0-11e9-9d8e-b845ff0c8cdc.png)

Adding a file key (value being 'atlassian.js') should fix it